.jenkins/driver_configs/baroclinic_c48_6ranks_dycore_only.yaml

```diff
@@ -0,0 +1,85 @@
+dycore_only: true
```

Comment:
we should probably add these as files to be tested here:
https://github.com/ai2cm/pace/blob/main/tests/main/driver/test_example_configs.py